config/locales/api/en.yml

```diff
@@ -445,14 +445,37 @@ en:
         cohort_title: "Result from last visit in %{period}"
         cohort_subtitle: "%{number_of_patients} %{patient_or_patients} registered in %{registration_period}"
         cohort_bar_tooltip: "%{total_patients} %{patient_or_patients} with %{threshold}"
-  monthly_screening_reports:
+  questionnaire_layout:
     male: "Male"
```

Comment:
I suppose transifex will try to update this key again when this PR is merged. We _might_ have to update the key name on transifex itself later.

Reply:
Since I have modified this key across all files, Transifex should update it in their sync. If they don't I'll do a manual sync/edit.